nixos/roles/webproxy.nix

```diff
@@ -63,9 +63,29 @@ in
           command = "${cfg.package}/bin/varnishadm status";
           timeout = 180;
         };
-        varnish_http = {
-          notification = "varnish port 8008 HTTP response";
-          command = "check_http -H localhost -p 8008 -c 10 -w 3 -t 20 -e HTTP";
+        varnish_http = let
+          # the plattform module passes multiple addresses in the form of '<address>:<port> -a <address>:<port> -a <address>:<port>'
```

Comment:
The host platform module also just provides a string field, so it depends on what the user enters there.

What you've written is one approach of how to pass multiple addresses, but according to [varnish docs](http://varnish-cache.org/docs/7.5/users-guide/command-line.html) another one is comma-separation of addresses like `-a '0.0.0.0:8080,[::]:8081'`. This needs to be taken into consideration as well when processing the string, quotation marks may be involved as well.